examples/seismic/acoustic/operators.py

```diff
@@ -40,15 +40,19 @@ def iso_stencil(field, m, s, damp, kernel, **kwargs):
     ----------
     field : TimeFunction
         The computed solution.
-    m : float
+    m : Function or float
         Square slowness.
     s : float or Scalar
         The time dimension spacing.
     damp : Function
         ABC dampening field.
-    kwargs : dict
-        Forward/backward wave equation (sign of u.dt will change accordingly
-        as well as the updated time-step (u.forwad or u.backward).
+    forward : boolean
+        The propagation direction. Defaults to True.
+    q : 
```

Comment:
TimeFunction, Function (time-independent) or float 
Full-space/time source of the wave-equation

Reply:
fixed